src/main/java/net/sf/rails/game/GameManager.java

```diff
@@ -982,10 +982,11 @@ protected boolean reload(GameAction reloadAction) {
 
         // Check action identity
         int index = 0;
+        int executedCnt = executedActions.size(); // save off the current # of executed actions as it will grow as we execute newly loaded
```

Comment:
Please add the comment above the this line. This will prevent formatters from cutting it up in multiple lines.

Reply:
Correctly configured formatters should never chop down comments :) But will move the comment above

Reply:
I think this is exactly what happens if you apply the IntelliJ or Eclipse formatter with default settings. I think it is quite likely that someone (like me) applies such a formatter on the code

Reply:
Comments on separate lines would be my preference as well sorry :)